src/pages/index.js

```diff
@@ -5,16 +5,30 @@ import * as sections from "../components/sections"
 import Fallback from "../components/fallback"
 import SEOHead from "../components/head"
 
+const Block = React.memo(({ block }) => {
```

Comment:
By using `React.memo`, the component will only re-render if it's props change. And without re-rendering, there is no opportunity for the suspense fallback component to show.